src/index.tsx

```diff
@@ -86,27 +86,29 @@ const wallets = [
   }),
 ];
 
-ReactDOM.render(
-  <React.StrictMode>
-    <StatsigProvider
-      sdkKey={
-        process.env.REACT_APP_STATSIG_SDK_KEY ||
-        "client-gQ2Zhz3hNYRf6CSVaczkQcZfK0yUBv5ln42yCDzTwbr"
-      }
-      waitForInitialization={true}
-      options={{
-        environment: {tier: process.env.NODE_ENV},
-      }}
-      user={{}}
-    >
-      <QueryClientProvider client={queryClient}>
-        <AptosWalletAdapterProvider plugins={wallets} autoConnect={true}>
-          <BrowserRouter>
-            <ExplorerRoutes />
-          </BrowserRouter>
-        </AptosWalletAdapterProvider>
-      </QueryClientProvider>
-    </StatsigProvider>
-  </React.StrictMode>,
-  document.getElementById("root"),
-);
+window.addEventListener("load", () => {
```

Comment:
please add a code comment that this is a temp fix, ideally with a link to the corresponding slack conversation and if applicable a linear ticket to address it.

Reply:
Yes please! Linear ticket for my side would be the most helpful